reader.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# author: nitepone
+"""
+Basic command parser to read custom "Perkins Script" files and execute
+appropriate commands from the associated adressable RGB strip control library.
+
+:docformat: reStructuredText
+"""
+
+def main():
+    filepath = input("Filepath of Perkins Script: ")
+    readScript(filepath)
+
+
+def lineParser(line):
```

Comment:
Should we consider using `str.split()`?

Reply:
This method is used to allow us to accept string arguments in the future if needed.